File: PHP/Mahasiswa.php
<?php

    class Mahasiswa{
        private $NIM = '';
        private $nama = '';
        private $jenis_kelamin = '';
        private $fakultas = '';
        private $prodi = '';

        public function __construct($NIM = '', $nama = '', $jenis_kelamin = '',
                        $fakultas = '', $prodi = ''){
            $this->NIM = $NIM;
            $this->nama = $nama;
            $this->jenis_kelamin = $jenis_kelamin;
            $this->fakultas = $fakultas;
            $this->prodi = $prodi;
        }

        public function set_NIM($NIM){
            $this->NIM = $NIM;
        }
        public function get_NIM(){
            return $this->NIM;
        }

        public function set_nama($nama){
            $this->nama = $nama;
        }
        public function get_nama(){
            return $this->nama;
        }

        public function set_jenis_kelamin($jenis_kelamin){
            $this->jenis_kelamin = $jenis_kelamin;
        }
        public function get_jenis_kelamin(){
            return $this->jenis_kelamin;
        }

        public function set_fakultas($fakultas){
            $this->fakultas = $fakultas;
        }
        public function get_fakultas(){
            return $this->fakultas;
        }

        public function set_prodi($prodi){
            $this->prodi = $prodi;
        }
        public function get_prodi(){
            return $this->prodi;
        }
    }


?>

File: C++/Mahasiswa.cpp
#include <bits/stdc++.h>
#include "SivitasAkademik.cpp"

using namespace std;

class Mahasiswa : public SivitasAkademik{
    
    private:
        string NIM;
        string nama;
        string jenis_kelamin;
        string fakultas;
        string prodi;

    public:
        Mahasiswa(){
            this->NIM = "";
            this->nama = "";
            this->jenis_kelamin = "";
            this->fakultas = "";
            this->prodi = "";
        }

        void set_NIM(string NIM){
            this->NIM = NIM;
        }

        string get_NIM(){
            return this->NIM;
        }

        void set_nama(string nama){
            this->nama = nama;
        }

        string get_nama(){
            return this->nama;
        }

        void set_jenis_kelamin(string jenis_kelamin){
            this->jenis_kelamin = jenis_kelamin;
        }

        string get_jenis_kelamin(){
            return this->jenis_kelamin;
        }

        void set_fakultas(string fakultas){
            this->fakultas = fakultas;
        }

        string get_fakultas(){
            return this->fakultas;
        }

        void set_prodi(string prodi){
            this->prodi = prodi;
        }

        string get_prodi(){
            return this->prodi;
        }
};